frontend/src/app/(navfooter)/u/[username]/page.tsx

```diff
@@ -6,9 +6,10 @@ import Profile from "@/components/user/Profile";
 import { get } from "@/lib/api/request";
 import type { User } from "@/lib/api/types";
 
-export async function generateMetadata({
-    params,
-}: { params: { username: string } }): Promise<Metadata> {
+export async function generateMetadata(props: {
+    params: Promise<{ username: string }>;
+}): Promise<Metadata> {
+    const params = await props.params;
```

Comment:
This is an RSC, right? So use(props.params) is better

Reply:
NVM this is a Next thing not a component. But the below guys should be changed I think

Reply:
next did this stuff automagically as part of the update, should they be await'd or use'd?